nncf/common/quantization/quantizer_removal.py

```diff
@@ -79,3 +91,53 @@ def _parse_node_relatives(node: NNCFNode, is_parents: bool):
             _parse_node_relatives(to_see_parents.pop(), is_parents=True)
 
     return to_cut, list(ops_to_return_in_orig_prec)
+
+
+def revert_operations_to_floating_point_precision(
+        operations: List[NNCFNode],
+        quantizers: List[NNCFNode],
+        quantized_model: TModel,
+        quantized_model_graph: NNCFGraph,
+        create_command_to_remove_quantizer_fn: Callable[[NNCFNode], TransformationCommand],
+        create_command_to_update_bias_fn: Callable[[NNCFNode, Any, NNCFGraph], TransformationCommand],
+        create_command_to_update_weight_fn: Callable[[NNCFNode, Any], TransformationCommand]) -> TModel:
```

Comment:
It looks like as utility function but not like standalone solution. What do you think about implementing `revert_operations_to_floating_point_precision` using the backend class. I mean introduce backed class with implementation of the specific functions such as 

```
create_command_to_remove_quantizer_fn: Callable[[NNCFNode], TransformationCommand],
create_command_to_update_bias_fn: Callable[[NNCFNode, Any, NNCFGraph], TransformationCommand],
create_command_to_update_weight_fn: Callable[[NNCFNode, Any], TransformationCommand])
```

or introduce dispatching functions.